README.md

```diff
@@ -4,5 +4,5 @@ Homebrew formulae for MediaArea.net GUI applications.
 
 How to install
 --------------------------
-First run `brew update`. Then to tap these formulae in homebrew run `brew tap mediaarea/mediaarea` and `brew install <formula>` to install from the formulae.
+First run `brew update`. Then to tap these formulae in homebrew run `brew tap MediaArea/mediaarea` and `brew install <formula>` to install from the formulae.
```

Comment:
having lowerwase and uppercase together on the same command line is source of problems.
Is it case insensitive?
is it possible to keep "mediaarea/mediaarea" here?